app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
avoid using rails generators to keep your code base small and get the practice in